<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bn_guiClass</class>
 <widget class="QMainWindow" name="bn_guiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>884</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>bn_gui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1043</width>
         <height>583</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="ClickableLabel" name="first_pic">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>File -&gt; Open first image</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ClickableLabel" name="second_pic">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>File -&gt; Open second image</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Frames:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox">
        <property name="minimum">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Add Line</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Remove Line</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Morph</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1065</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_first_image"/>
    <addaction name="actionOpen_second_image"/>
    <addaction name="actionOpen_Line_File"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionReadme"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_first_image">
   <property name="text">
    <string>Open first image</string>
   </property>
  </action>
  <action name="actionOpen_second_image">
   <property name="text">
    <string>Open second image</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionReadme">
   <property name="text">
    <string>Readme</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen_Line_File">
   <property name="text">
    <string>Open Line File</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>ClickableLabel.h</header>
   <slots>
    <signal>clicked(QPoint)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="bn_gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>bn_guiClass</receiver>
   <slot>remove_line()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>641</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_second_image</sender>
   <signal>triggered()</signal>
   <receiver>bn_guiClass</receiver>
   <slot>open_second()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_first_image</sender>
   <signal>triggered()</signal>
   <receiver>bn_guiClass</receiver>
   <slot>open_first()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>bn_guiClass</receiver>
   <slot>set_output_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>734</x>
     <y>641</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>-6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>first_pic</sender>
   <signal>clicked(QPoint)</signal>
   <receiver>bn_guiClass</receiver>
   <slot>first_image_clicked(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_Line_File</sender>
   <signal>triggered()</signal>
   <receiver>bn_guiClass</receiver>
   <slot>load_lines()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>second_pic</sender>
   <signal>clicked(QPoint)</signal>
   <receiver>bn_guiClass</receiver>
   <slot>second_image_clicked(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>742</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>bn_guiClass</receiver>
   <slot>add_line()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>646</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>bn_guiClass</receiver>
   <slot>start_morph()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>805</x>
     <y>646</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>bn_guiClass</receiver>
   <slot>highlight_list(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>760</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open_first()</slot>
  <slot>open_second()</slot>
  <slot>remove_line()</slot>
  <slot>set_output_file()</slot>
  <slot>first_image_clicked(QPoint)</slot>
  <slot>second_image_clicked(QPoint)</slot>
  <slot>load_lines()</slot>
  <slot>add_line()</slot>
  <slot>start_morph()</slot>
  <slot>highlight_list(int)</slot>
 </slots>
</ui>
